pkg/utils/github_utils.go

```diff
@@ -0,0 +1,26 @@
+package utils
+
+import (
+	"context"
+
+	"github.com/google/go-github/v59/github"
+)
+
+// GetLatestGitHubRepoRelease returns the latest release tag for a GitHub repository
+func GetLatestGitHubRepoRelease(owner string, repo string) (string, error) {
```

Comment:
Can this be disabled via atmos config? e.g. in air-gapped environments, this will not work. More and more enterprises are using atmos, including in restricted environments. We want to make sure it doesn't break for them.

Reply:
What's the timeout? e.g. if GitHub is down, we don't want atmos to fail. Also, we can fail fast. If it times out, just hide the dialog.

Reply:
what's the timeout? We should set it very short ~ 1000ms (or less)

Reply:
Is it only checking when you run `atmos version`? If so that's okay. 

In the near future, we may want to consider running the check once a day any any command run.